sources/packages/backend/libs/test-utils/src/factories/application.ts

```diff
@@ -29,3 +48,98 @@ export function createFakeApplication(relations?: {
   application.applicationNumber = faker.random.alphaNumeric(10);
   return application;
 }
+
+/**
+ * Create and save to the database an application with disbursement record(s) with all the dependencies.
+ * @param dataSource manages the repositories to save the data.
+ * @param relations dependencies.
+ * - `institution` related institution.
+ * - `institutionLocation` related location.
+ * - `disbursementValues` related disbursement schedules.
+ * - `student` related student.
+ * @param options additional options:
+ * - `createSecondDisbursement` if provided and true creates a second disbursement,
+ * otherwise only one disbursement will be created.
+ * @returns the created application and its dependencies including the disbursement
+ * with the confirmation of enrollment data.
+ */
+export async function saveFakeApplicationDisbursements(
```

Comment:
do we need to include the application status in function name too, as disbursements are created before the "enrollment" state

Reply:
The `saveFakeApplicationDisbursements` was the already existing method `saveFakeApplicationCOE` and the intention was to avoid all the work to have a student application created with the disbursement records associated.
Including the status in the name would not be recommended, IMO, since the idea would be to use this method as a more generic one. Since it is a `saveFake` and not a `createFake` we may consider adding more parameters that would be avoiding an additional saving to happen on the tests themselves (which we have been avoiding for `createFake` methods). Thoughts?
@dheepak-aot @andrepestana-aot @guru-aot @sh16011993

Reply:
As per the Team decision and as per mentioned in the above comment the method `saveFakeApplicationDisbursements` is now receiving the application status in the options and adjusting other properties. It causes some minor impact along with other files but I believe that it will not disturb the PR in general.